Sources/BrowserServicesKit/PrivacyConfig/Features/PrivacyFeature.swift

```diff
@@ -64,6 +65,15 @@ public enum AutofillSubfeature: String, PrivacySubfeature {
     case autofillPasswordGeneration
 }
 
+public enum NetworkProtectionSubfeature: String, Equatable, PrivacySubfeature {
+    public var parent: PrivacyFeature {
+        .networkProtection
+    }
+
+    case waitlist
```

Comment:
I'm also adding the waitlist subfeature so we can save from having to do an extra PR.

Reply:
Nice 👏